fe/fe-core/src/main/java/com/starrocks/credential/aliyun/AliyunCloudCredential.java

```diff
@@ -39,8 +39,6 @@ public AliyunCloudCredential(String accessKey, String secretKey, String endpoint
 
     @Override
     public void applyToConfiguration(Configuration configuration) {
-        configuration.set("fs.oss.impl", "com.aliyun.jindodata.oss.JindoOssFileSystem");
```

Comment:
you don't need to change code here, our program will not go there.

Reply:
OK. I will put it back.